drivers/sensor/tdk/icp101xx/Message.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2024 TDK Invensense
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _INV_MESSAGE_H_
+#define _INV_MESSAGE_H_
+
+/* Disable Message from the driver */
+#define INV_MSG(...)
```

Comment:
This is a bit confusing, each driver has a log level adjustable already so this should not be needed

Reply:
The goal of his PR is to make sure Zephyr driver for ICP101xx sensors is using the official driver from TDK.
This code is part of this official TDK driver and cannot be modified for Zephyr.